vault/provider.go

```diff
@@ -41,6 +41,16 @@ const (
 // The key of the mutex should be the path in Vault.
 var vaultMutexKV = helper.NewMutexKV()
 
+var (
+	VaultVersion110 *version.Version
+	VaultVersion111 *version.Version
+)
+
+func init() {
+	VaultVersion110 = version.Must(version.NewSemver(consts.VaultVersion10))
```

Comment:
nit: update constant name to match. Include the major.

Reply:
Updated